### PR TITLE
Updating metadata

### PR DESCRIPTION
**Pull request by terrimorton**

### CODE REVIEW

**Comment by shirhatti** [comment]
I forget off the top of my head, but what are the recommended character limits for title and description according to our SEO guidelines? I'm afraid this might be over.

File: iis/get-started/whats-new-in-iis-10/thread-pool-ideal-cpu-numa-optimization.md [hunk 1]
```diff
@@ -1,8 +1,12 @@
 ---
 title: "IIS Thread Pool Ideal CPU Optimization for NUMA hardware | Microsoft Docs"
 author: bangbingsyb
-description: ""
+description: "The article introduces native HTTP Strict Transport Security (HSTS) support in IIS 10.0 version 1709."
```
Comment (bangbingsyb) [inline iis/get-started/whats-new-in-iis-10/thread-pool-ideal-cpu-numa-optimization.md]: Why the HSTS description was added in this article?

Comment (terrimorton) [reply iis/get-started/whats-new-in-iis-10/thread-pool-ideal-cpu-numa-optimization.md]: Copy and paste error. Please update as appropriate.

Comment (terrimorton) [reply iis/get-started/whats-new-in-iis-10/thread-pool-ideal-cpu-numa-optimization.md]: I have gone ahead and updated the description in the Master branch

**Comment by terrimorton** [comment]
@shirhatti Our CATS guidelines for the Title is 95 or less. The SEO team told us only that it "should not be too long". There is no guideline for the Description length in CATS, but the SEO team told us it should be 115-145 characters.

**Pull request merged**